.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.git
```

Comment:
I see. So I'll have to be careful not to deploy code without package.json set up correctly.

(I'm not currently using docker, but may switch very soon, which is why these questions are on my mind ☺)